torch/testing/_internal/common_utils.py

```diff
@@ -1728,6 +1728,10 @@ def random_sparse_matrix(rows, columns, density=0.01, **kwargs):
     dtype = kwargs.get('dtype', torch.double)
     device = kwargs.get('device', 'cpu')
     singular = kwargs.get("singular", False)
+    values_domain = kwargs.get("domain", None)
```

Comment:
Why is this function **kwargs? 

We don't need to fix random_sparse_matric() in this PR but that seems odd, and it makes documenting its supported kwargs tricky.